src/controllers/dynakube/activegate/internal/statefulset/statefulset.go

```diff
@@ -94,6 +95,10 @@ func (statefulSetBuilder StatefulSetBuilder) addLabels(sts *appsv1.StatefulSet)
 	sts.Spec.Template.ObjectMeta.Labels = kubeobjects.MergeMap(statefulSetBuilder.capability.Properties().Labels, appLabels.BuildLabels())
 }
 
+func (statefulSetBuilder StatefulSetBuilder) addAnnotations(sts *appsv1.StatefulSet) {
+	sts.Spec.Template.ObjectMeta.Annotations = kubeobjects.MergeMap(sts.Spec.Template.ObjectMeta.Annotations, statefulSetBuilder.dynakube.Spec.ActiveGate.Annotations)
+}
+
```

Comment:
```suggestion
func (statefulSetBuilder StatefulSetBuilder) addUserAnnotations(sts *appsv1.StatefulSet) {
	sts.ObjectMeta.Annotations = kubeobjects.MergeMap(sts.ObjectMeta.Annotations, statefulSetBuilder.dynakube.Spec.ActiveGate.Annotations)
	sts.Spec.Template.ObjectMeta.Annotations = kubeobjects.MergeMap(sts.Spec.Template.ObjectMeta.Annotations, statefulSetBuilder.dynakube.Spec.ActiveGate.Annotations)
}
```
maybe we should add the annotations everywhere ? (on the statefulset and the pod)